composer.json

```diff
@@ -36,6 +36,7 @@
         "doctrine/orm": "~2.3",
         "symfony/yaml": "~2.2|~3.0",
         "symfony/validator": "~2.2|~3.0",
+        "symfony/property-info": "~2.8|~3.0",
```

Comment:
Finally removed as asked by @stof (I also think it doesn't add any help to suggest this component).